app.js

```diff
@@ -1,8 +1,98 @@
 // import functions and grab DOM elements
+const btnOne = document.getElementById('cupOneBtn');
+const btnTwo = document.getElementById('cupTwoBtn');
+const btnThree = document.getElementById('cupThreeBtn');
 
+const resetBtn = document.getElementById('resetButton');
+
+const cupOne = document.getElementById('cup-one');
+const cupTwo = document.getElementById('cup-two');
+const cupThree = document.getElementById('cup-three');
+
+const correctEl = document.getElementById('correct');
+const incorrectEl = document.getElementById('incorrect');
+const totalEl = document.getElementById('total');
+
+const cups = [
+    0, 1, 2
+];
 // let state
+let correct = 0;
+let total = 0;
+
+function resetCups() {
+    cupOne.src = './assets/cupgame1.png';
+    cupTwo.src = './assets/cupgame1.png';
+    cupThree.src = './assets/cupgame1.png';
+}
+
+function statusDisplay() {
+    correctEl.textContent = correct;
+    totalEl.textContent = total;
+    incorrectEl.textContent = total - correct;
+}
+function resetStatus() {
+    correct = '0';
```

Comment:
Maybe consider calling `resetCups()` in here also just so the images reset when the reset button gets clicked?